_data/investing.yml

```diff
@@ -33,6 +33,12 @@ websites:
       exceptions:
           text: "2FA only available on investment accounts. Not available for 401K plans."
 
+    - name: HealthEquity
+      url: https://www.healthequity.com/
```

Comment:
I can get to https://www.healthequity.com from the US, but it's just a login page.  Interestingly enough, connecting over http gets you to their home page, the https gets you directly to the login page. So perhaps just switch to http?